src/commands/cancel.ts

```diff
@@ -0,0 +1,17 @@
+import Command from '../@types/Command'
+
+const cancelShortcut: Command = {
+  id: 'cancel',
+  label: 'Cancel',
+  description: 'Cancel the current gesture.',
+  gesture: undefined,
+  keyboard: { key: ',', meta: true },
```

Comment:
I'm not sure why there is a keyboard command assigned. Maybe it was used for testing?